test/svg-sprite/minimal-configuration/rerun.test.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+/* eslint-disable no-unused-expressions, max-nested-callbacks */
+const path = require('path');
+const glob = require('glob');
+const should = require('should');
```

Comment:
BTW I planned at some point to get rid of should and just use assert, but never got to do it :)

Reply:
Yeah it will be eliminated in jest version of tests :)